src/main/java/year2017/day5/Main.java

```diff
@@ -0,0 +1,69 @@
+package year2017.day5;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+        List<Integer> instructions = new ArrayList<>();
+        List<Integer> instructionsPart2 = new ArrayList<>();
+        // Read input
+        try (BufferedReader reader = new BufferedReader(new FileReader("src/main/resources/inputs/day_05_2017.txt"))) {
+            String line;
+
+            while ((line = reader.readLine()) != null) {
+                int instruction = Integer.parseInt(line);
+                instructions.add(instruction);
+                instructionsPart2.add(instruction);
+            }
+
+            // Read input[i] = n    countStep += 1
+            // If i+1 <= input.size
+            int countStep = 0;
+            for (int i = 0; i < instructions.size(); ) {
```

Comment:
Peut-être qu'un while serait plus adapté, si tu gères l'index dans le corps du `for` ?